README.md

```diff
@@ -1,30 +1,30 @@
 # Couchbase Lite for VSCode
 
-This is a Visual Studio Code extension for interacting with a Couchbase Lite database.  This includes enumerating documents and running ad-hoc queries.  It is heavily inspired by the [SQLite extension](https://github.com/AlexCovizzi/vscode-sqlite/) that provides similar functionality.
-
-This repo can also be used as a standalone node.js binding for Couchbase Lite for C.  To do so, simply follow the normal process of building as described below.  The file *out/binding/binding.js* can be imported into a node.js application, with the caveat that you must also copy the cblite shared binary (i.e. libcblite.so.3 libcblite.dylib or cblite.dll) into *build/Release* (next to cblite-js.node).
+This is a Visual Studio Code extension for interacting with a Couchbase Lite database. [Couchbase Lite](https://docs.couchbase.com/couchbase-lite/3.0/index.html) is full featured NoSQL JSON document database for mobile, desktop and embedded platforms. This extension includes support for enumerating documents and running ad-hoc queries.  Th plugin is heavily inspired by the [SQLite extension](https://github.com/AlexCovizzi/vscode-sqlite/) that provides similar functionality.
 
 ## Quick Tour
 
 ### Open a Database
 
-This can be accomplished in a few different ways.  The fastest way is to have Visual Studio Code open with a workspace that contains some `cblite2` databases inside.  Then you can right click and open as in the following:
+This can be accomplished in a few different ways. 
+- The fastest way is to have Visual Studio Code open with a workspace that contains some `cblite2` databases inside.  Then you can right click and open as in the following:
 
 <img src="resources/readme/open_rightclick.gif" alt="Open Database With Right Click" />
 
-If you have opened a folder as described above you can also use the *Open Database* command from the command pallete, and any cblite2 databases discovered will be listed, with a final entry (*Choose database from file*) to open using a standard file open dialog.
+- If you have opened a folder as described above you can also use the *Open Database* command from the [command pallete](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette), and any cblite2 databases discovered will be listed, with a final entry (*Choose database from file*) to open using a standard file open dialog.
 
 <img src="resources/readme/open_pallete.gif" width=500 alt="Open Database With Command Pallete" />
 
-:warning: Opening a database using this plugin will automatically upgrade it silently.  If this is undesirable, make a copy of the database folder before opening it with the plugin.
+
+:**NOTE**: Opening a database using this plugin will automatically upgrade it to 3.0 version of Couchbase Lite silently.  If this is undesirable, make a copy of the database folder before opening it with the plugin.
```

Comment:
Why the change from warning to note?  Leaving it as the warning icon will use an icon in GitHub markdown.